dashboard/src/app/workspaces/workspace-details/workspace-machine-agents/machine-agents.html

```diff
@@ -31,6 +31,8 @@
             <div flex="100"
                  layout="row"
                  layout-align="start stretch"
+                 agent-item-name="{{agentItem.name}}"
```

Comment:
what are these div attributes ?

Reply:
From **agent-item-name** there is access to installer's state, name, version and description. 
![selection_084](https://user-images.githubusercontent.com/7760565/33556781-c7f050fe-d90d-11e7-9a67-101358dc1c1a.png)
And **agent-item-version** need for switching installers with different versions(in this case are two versions of the **Workspace API**).